FLEDGE.md

```diff
@@ -1255,6 +1257,20 @@ If the top-level auction has a `sellerCurrency` configured, this will be its cur
     *   "language-exclusions"
     *   "category-exclusions"
 
+#### 7.1.2 Downsampling
+
+This design has three main goals:
+*   Prevent sending fDO reports very often to protect user privacy:
```

Comment:
```suggestion
* Prevent sending fDO reports very often to protect user privacy:
```

Reply:
(Oops sorry, I think this will make the next three bullets properly nest under this one)